pages/common/gem.md

```diff
@@ -19,6 +19,10 @@
 
 `gem install {{gemname}} --version {{1.0.0}}`
 
+- Install the latest matching (SemVer) version of a gem:
+
+`gem install {{gemname}} --version '~> {{1.0}}'`
```

Comment:
Sorry, but we can't accept more than 8 examples per page.

Reply:
Is there a less important example we could remove?

Reply:
`- Search for remote gem(s):` and `gem search {{regexp}} --all` are quite equal, so drop the first in favor of the latter?

Reply:
Hey @sbrl, Do we need `snake_case` for this variable?
```suggestion
`gem install {{gem_name}} --version '~> {{1.0}}'`
```